models/intel/person-reidentification-retail-0265/model.yml

```diff
@@ -16,29 +16,29 @@ description: >-
+  - name: FP32/person-reidentification-retail-0265.bin
     size: 21059332
     sha256: 396b43fe9ae7c10bb5b8e5b8b5a0d024f56b51b64561fe81e55053b5df01b6a3
-    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0300/FP32/person-reidentification-retail-0300.bin
-  - name: FP16/person-reidentification-retail-0300.xml
+    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0265/FP32/person-reidentification-retail-0265.bin
+  - name: FP16/person-reidentification-retail-0265.xml
     size: 447699
     sha256: 19f0414d4a3c9e271731caa02d9a158b95d529a0c7c8f13c8c090a9af90fb5f0
-    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0300/FP16/person-reidentification-retail-0300.xml
-  - name: FP16/person-reidentification-retail-0300.bin
+    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0265/FP16/person-reidentification-retail-0265.xml
+  - name: FP16/person-reidentification-retail-0265.bin
     size: 10529712
     sha256: 98ab444ac49800a07c59d4ca121f641b3f3b737451e27b675b6468dbf0450425
-    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0300/FP16/person-reidentification-retail-0300.bin
-  - name: FP16-INT8/person-reidentification-retail-0300.xml
+    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0265/FP16/person-reidentification-retail-0265.bin
+  - name: FP16-INT8/person-reidentification-retail-0265.xml
     size: 1362674
     sha256: 7f99cf664f251e2bafa7b64e5c814e00c809568adc57258c28dd27c82b946b10
-    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0300/FP16-INT8/person-reidentification-retail-0300.xml
-  - name: FP16-INT8/person-reidentification-retail-0300.bin
+    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0265/FP16-INT8/person-reidentification-retail-0265.xml
+  - name: FP16-INT8/person-reidentification-retail-0265.bin
     size: 7743940
     sha256: 768bc7d22d4dfbad8d7e9ce793fa5ea6a2185003793ce780ae0d83734f8dd2d0
-    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0300/FP16-INT8/person-reidentification-retail-0300.bin
+    source: https://download.01.org/opencv/2020/openvinotoolkit/2020.3/open_model_zoo/models_bin/1/person-reidentification-retail-0265/FP16-INT8/person-reidentification-retail-0265.bin
```

Comment:
@DmitriySidnev , I am not sure that you need to provide `model.yml`, these files generated with models package update. 
Also obviously links are invalid because directory with models package for openvino R3 does not contain such models.

Reply:
`model.yml` removed.